src/bigfix/Computers/ComputerResults.cs

```diff
@@ -9,6 +10,11 @@ public ComputerResult(ComputerResponse response)
         }
 
         public string Get(string prop) {
+            var temp = AllProperties.Where(r => r.Name.Equals(prop, System.StringComparison.OrdinalIgnoreCase)).ToList();
+            if (temp.Count > 1)
+            {
+                return String.Join(", ", temp.Select(a => a.Value).Distinct().ToArray());
+            }
```

Comment:
How about instead, just this:
`string.join(", ", AllProperties.Where(r => r.Name.Equals(prop, System.StringComparison.OrdinalIgnoreCase)));`